### ISSUE TITLE
Documentation for Phoenix

### ISSUE THREAD

**Issue by cloudbring** [ride/exrm-heroku#1]
![](https://raw.githubusercontent.com/phoenixframework/phoenix/master/priv/static/phoenix.png)
It would be great if there was a way to deploy `Phoenix`apps (https://github.com/phoenixframework/phoenix) to Heroku with `exrm-heroku`.
- Has anyone deployed one to heroku with `exrm-heroku`?

**Comment by sheharyarn** [comment]
Bump!

**Comment by sheharyarn** [comment]
Try this instead: [__gjaldon / phoenix-static-buildpack__](https://github.com/gjaldon/phoenix-static-buildpack)

**Comment by guilleiguaran** [comment]
Yes, we have deployed a couple of multi-environment apps using this, we will publish some documentation for Phoenix soon

**Comment by arjan** [comment]
"soon"? :-)
